documentation/asciidoc/accessories/iqaudio/codec_zero.adoc

```diff
@@ -1,17 +1,17 @@
 === Raspberry Pi Codec Zero
 
-Raspberry Pi Codec Zero is a Raspberry Pi Zero sized audio HAT. It delivers bi-directional digital audio signals (I2S) between Raspberry Pi Zero and its onboard Dialog Semiconductor DA7212 codec. Codec Zero supports a range of input and output devices,
+Raspberry Pi Codec Zero is a Raspberry Pi Zero sized audio HAT (Hardware Attached on Top). It delivers bi-directional digital audio signals (I2S) between Raspberry Pi Zero and its onboard Dialog Semiconductor DA7212 codec. Codec Zero supports a range of input and output devices.
 
 * High performance 24-bit audio codec
 * Supports common audio sample rates between 8-96kHz
-* Built in MEMS microphone (Mic2)
+* Built in MEMS (microelectromechanical) microphone (Mic2)
```

Comment:
Should probably capitalise this as MicroElectroMechanical to make the acronym more obvious? :shrug: (but that's probably a question for @helenlynn )
Although https://en.wikipedia.org/wiki/Microelectromechanical_systems suggests that "micro-electro-mechanical systems" is a valid alternative.

Reply:
I'd say leave it as it is. Camel case and hyphens would be obtrusive clutter here, and the acronym is probably not too hard to spot without them for most people.